widgets/nuxeo-date.html

```diff
@@ -22,7 +22,7 @@
 
 <dom-module id="nuxeo-date">
   <template>
-    <span id="datetime" hidden$="[[!datetime]]">[[formatDate(datetime, format)]]</span>
+    <span id="datetime" hidden$="[[!datetime]]">[[formatDateTime(datetime, format)]]</span>
```

Comment:
We should not change this since we will break compatibility.

Reply:
sorry I copied this from the other version and didn't notice it was different, will revert it.